contrib/ansible/roles/flannel-config/templates/config-flannel.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+etcdctl mk /coreos.com/network/config '{"Network": "{{ flannel_network }}/16", "SubnetLen": 24, "Backend": {"Type": "vxlan"}}'
```

Comment:
should we make the /16 and 24 configurable?  I have an alternate flannel implementation i haven't pushed.  Maybe we should pull this out and work on flannel in a different PR?